docs/src/validation_services.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Validation Services
+---
+
+**Subject to change.**
+
+Running a Solana validation-client required relatively modest upfront hardware capital investment. **Table 2** provides an example hardware configuration to support ~1M tx/s with estimated ‘off-the-shelf’ costs:
+
+| Component         | Example                                          | Estimated Cost |
+| :---------------- | :----------------------------------------------- | :------------- |
+| GPU               | 2x 2080 Ti                                       | \$2500         |
+| or                | 4x 1080 Ti                                       | \$2800         |
+| OS/Ledger Storage | Samsung 860 Evo 2TB                              | \$370          |
+| Accounts storage  | 2x Samsung 970 Pro M.2 512GB                     | \$340          |
+| RAM               | 32 Gb                                            | \$300          |
```

Comment:
64 GB is bare minimum these days, are lot of nodes are already running on 128 GB RAM